http/session/error.go

```diff
@@ -3,7 +3,6 @@ package session
 import "errors"
 
 var (
-	ErrFailedConfig = errors.New("failed config")
-	ErrNotValid     = errors.New("not valid")
-	ErrNoUser       = errors.New("no user")
+	ErrNotValid = errors.New("not valid")
```

Comment:
Put these in the root rather than having errors defined per package?

Reply:
I'm a bit undecided on the utility of having the root repo providing all sentinel errors for every other package. In line with that, the introduction of multiple error wrapping in Go 1.20 gives us opportunity to reflect on the strategy for errors in trails.